### ISSUE TITLE
Wow!

### ISSUE THREAD

**Issue by danieljamesross** [shers003/AlabnyCode#1]
So good to see such great code. Well done! You've put in loads of hard work and it really shows.

Some points that can be improved:

- Have default images for the carousel so that there are images right from the get-go
- Consider code splitting your different apps into separate pages. Having everything on one page can be difficult to navigate.
- Watch out for spelling, e.g. `Baisically!!`
- Replace the `OK` and `Cancel` buttons with more meaningfull options.
- Mission 5, add pics instead of text
- TicTacToe, try displaying whoever's "go" it is in a prettier way, i.e. not in a text box

**Comment by shers003** [comment]
Thank you Dan,
I will make these improvements. I really enjoyed making it lots of fun!
Thanks again :),Sonny 
    On Tuesday, 17 November 2020, 19:34:09 GMT, Dan <notifications@github.com> wrote:  
 
 


So good to see such great code. Well done! You've put in loads of hard work and it really shows.

Some points that can be improved:
   
   - Have default images for the carousel so that there are images right from the get-go
   - Consider code splitting your different apps into separate pages. Having everything on one page can be difficult to navigate.
   - Watch out for spelling, e.g. Baisically!!
   - Replace the OK and Cancel buttons with more meaningfull options.
   - Mission 5, add pics instead of text
   - TicTacToe, try displaying whoever's "go" it is in a prettier way, i.e. not in a text box

**Comment by shers003** [comment]
Oh just realised this was with Github lol

**Comment by danieljamesross** [comment]
Yeah, it's a great way to introduce you to how a professional dev works. Once you've made these changes then you can `close` the issue.

**Comment by shers003** [comment]
Hey Dan, I have done everything just not sure what the 'ok' and 'cancel' buttons are? Actually I just realised it was the difficulty setting, I'll sort it

**Comment by danieljamesross** [comment]
"Mission 5 task"
